dell-csi-helm-installer/csi-offline-bundle.md

```diff
@@ -2,8 +2,7 @@
 
 ## Description
 
-The `csi-offline-bundle.sh` script can be used to create a package usable for offline installation of the Dell EMC CSI Storage Providers, via either Helm 
-or the Dell CSI Operator. 
+The `csi-offline-bundle.sh` script can be used to create a package usable for offline installation of the Dell EMC CSI Storage Providers, via Helm. 
```

Comment:
minor edit, change to : 
The `csi-offline-bundle.sh` script can be used to create a package for offline installation of Dell CSI storage providers for deployment via HELM.